react/wbooks/src/app/screens/SignUp/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18next from 'i18next';
+
+import FormWrapper from '../../components/Formik';
+import CustomField from '../../components/CustomField';
+import logoWolox from '../../assets/LogoWolox.png';
+
+import styles from './styles.module.scss';
+
+function SignUp({ initialValues, validationSchema, onSubmit }) {
+  return (
+    <FormWrapper initialValues={initialValues} validationSchema={validationSchema} handleSubmit={onSubmit}>
+      {({ handleSubmit, ...props }) => (
+        <div className={styles.formContainer}>
+          <img className={styles.logoImage} src={logoWolox} />
+          <form className={styles.formContent} onSubmit={handleSubmit}>
+            <CustomField type="text" name="name" id="Nombre" {...props} />
+            <CustomField type="text" name="lastName" id="Apellido" {...props} />
+            <CustomField type="email" name="email" id="Email" {...props} />
+            <CustomField type="password" name="password" id="Contraseña" {...props} />
+            <CustomField type="password" name="confirmPassword" id="Confirmar Contraseña" {...props} />
```

Comment:
Why are you adding all this id 🤔  Also... the id should be in english and snake case

Reply:
I added id becuase if the user press in label for example password, the screen active to write in input password.